### DISCUSSION TITLE
Make go to commands navigate instead of opening a new file explorer

### DISCUSSION THREAD

**Discussion by ChristopheL77** [moudey/Shell#267] [Q&A]
Hi, 

Firstly, thanks for this great tool :-)

Is there a way to open a folder in current file explorer like the standard open command in registry ?

I tried with and without cmd property
```
item(title='Windows1' cmd=explorer verb="Explore" dir=sys.dir)

item(title='Windows1' cmd=explorer verb="Open" dir=sys.dir)
```
but it didn't work.

Thanks

**Comment by moudey** [comment]
Do you mean to navigate in the same window?

**Reply by ChristopheL77** [reply]
Yes, in order to make a favorites folders menu.

**Comment by ChristopheL77** [comment]
Is this scenario supported ?

**Reply by moudey** [reply]
Work to support this.

**Reply by ChristopheL77** [reply]
Nice thanks :)

**Comment by moudey** [comment]
@ChristopheL77 try this [build 35](https://nilesoft.org/download/shell/debug.zip)
```
item(title="navigate" cmd=command.navigate('c:\windows\system32'))
```

**Reply by ChristopheL77** [reply]
Awesome, thanks a lot :)

I tried quickly but it seems you changed configuration files format, so I will have to test it later when I have more time.
